Microsoft.Bot.Builder.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.26730.12
+# Visual Studio Version 16

```

Comment:
Please don't change this file - consider removing it from your changeset.

Reply:
Looks like this has already been updated in master